examples/play.rs

```diff
@@ -24,14 +27,20 @@ async fn main() {
     let track = SpotifyId::from_base62(&args[3]).unwrap();
 
     let backend = audio_backend::find(None).unwrap();
+    let ditherer = dither::find_ditherer(None).unwrap();
+    let noise_shaper = shape_noise::find_noise_shaper(None).unwrap();
```

Comment:
Anyone who would see this example to implement their own simple player would get confused, I believe. Is it really necessary to give them such a prominent place in the API?

Reply:
To be clear, I think `PlayerConfig` would be a more appropriate place to configure things like this.